src/libraries/System.Runtime.InteropServices.JavaScript/src/System.Runtime.InteropServices.JavaScript.csproj

```diff
@@ -76,6 +76,11 @@
     <Compile Include="System\Runtime\InteropServices\JavaScript\JSWebWorker.cs" />
     <Compile Include="System\Runtime\InteropServices\JavaScript\JSSynchronizationContext.cs" />
     <Compile Include="System\Runtime\InteropServices\JavaScript\JSAsyncTaskScheduler.cs" />
+    <!-- Can't build System.Threading.Channels.csproj without these -->
```

Comment:
Why is that? This workaround does not look right.

Reply:
Thanks for looking, it was as though even though these transitive dependencies were built, they weren't found for this project.  However it doesn't seem consistent and I removed this change now.